.github/workflows/main-publish.yml

```
@@ -0,0 +1,32 @@
+# PyPi publish for dcicutils 
+
+name: publish 
+
+# Controls when the action will run. 
+on:
+
+  # Publish on all tags
+  push:
+    tags:
+    - '*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-18.04
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Publish
+        env:
+          PYPI_USER: ${{ secrets.PYPI_USER }}
+          PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: poetry publish --build --username=$PYPI_USER --password=$PYPI_PASSWORD 
```

I think probably we should prefer (untested):
```suggestion
        run: make publish
```
It does some additional checks, so you'll want to check that the checks it does are proper and will work on GA.